### DISCUSSION TITLE
How to disable web interface

### DISCUSSION THREAD

**Discussion by omgbox** [KiraLT/torrent-stream-server#226] [Q&A] [answered]
how can I disable the web interface but keep everything running behind because as I am interacting with the api. i don't want the frontend to be exposed to the public.

**Comment by omgbox** [comment]
i was reading the configure section 

inside config.json 

`i set  "frontendEnabled": false,`

but the frontend is still being built.

**Comment by KiraLT** [comment, marked as answer]
You can add the following to the `config.json`:

```json
{
    "security": {
        "frontendEnabled": false
    }
}
```

Or you can use environment variable:

`SECURITY__FRONTEND_ENABLED=false`

> You can find more info in [documentation](https://github.com/KiraLT/torrent-stream-server/wiki/configuration)

**Reply by omgbox** [reply]
thank you.
